--- origsrc/zathura-ps-0.2.2/config.mk	2013-05-12 16:29:11.000000000 -0500
+++ src/zathura-ps-0.2.2/config.mk	2013-07-30 13:38:29.161855900 -0500
@@ -29,13 +29,14 @@ GIRARA_INC ?= $(shell pkg-config --cflag
 GIRARA_LIB ?= $(shell pkg-config --libs girara-gtk${ZATHURA_GTK_VERSION})
 
 ZATHURA_INC ?= $(shell pkg-config --cflags zathura)
+ZATHURA_LIB ?= $(shell pkg-config --libs zathura)
 PLUGINDIR ?= $(shell pkg-config --variable=plugindir zathura)
 ifeq (,${PLUGINDIR})
 PLUGINDIR = ${LIBDIR}/zathura
 endif
 
 INCS = ${GLIB_INC} ${SPECTRE_INC} ${GIRARA_INC} ${ZATHURA_INC}
-LIBS = ${GLIB_LIB} ${SPECTRE_LIB} ${GIRARA_LIB}
+LIBS = ${GLIB_LIB} ${SPECTRE_LIB} ${GIRARA_LIB} ${ZATHURA_LIB}
 
 # flags
 CFLAGS += -std=c99 -fPIC -pedantic -Wall -Wno-format-zero-length $(INCS)
